InTune/InTune/Extensions/Designable.swift

```diff
@@ -73,3 +74,14 @@ class DesignableLabel: UILabel {
      layer.borderWidth = borderWidth
      layer.borderColor = borderColor
    }}
+
+extension UIImage {
+  static func resizeImage(originalImage: UIImage, rect: CGRect) -> UIImage {
+    let rect = AVMakeRect(aspectRatio: originalImage.size, insideRect: rect)
```

Comment:
can this whole imageRendering part be handled statically in the xibs? or with the help of some cell sizing. I this calling this drawFunction is unnecessary unless it can not be handled easily by what I said about cell sizing and handling in xibs.